contents/javascript/javascript.md

```diff
@@ -0,0 +1,114 @@
+# Writing Testable Javascript
+
+JavaScript is a powerful language, and because it is powerful, there are multiple ways to go about doing the same thing. The end result is that multiple collaborators working on a single project can produce different code that does the same thing. That is why there is need for a standard to writing JavaScript code, especially on the front end.
```

Comment:
> JavaScript is a powerful language, and because it is powerful, there are multiple ways to go about doing the same thing.

Reword this, it just sounds awkward.